source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -916,6 +917,22 @@ message TransformationFunctionCall {
 
 }
 
+message TransformationInvertComparisonOperator {
+
+  // For some instruction with result id |operator_id| that
+  // represents a binary operator (<, >, <=), this transformation
```

Comment:
Either give an exhaustive list or use e.g. to make it clear that these are just three examples.

Also: be more precise than "a binary operator", because +, -, etc., are binary operators.  I guess you mean a comparison operator specifically.

Reply:
Fixed in 682be74.